PracticeWebApp.FuncTest/Hooks/Connection.cs

```diff
@@ -0,0 +1,38 @@
+using BoDi;
+using PracticeWebApp.Tests.integrationTests;
+using TechTalk.SpecFlow;
+
+namespace PracticeWebApp.FuncTest.Hooks
+{
+    [Binding]
+    public sealed class Connection
+    {
+        private readonly IObjectContainer objectContainer;
+        private static CustomWebApplicationFactory<Program> _webApplicationFactory = null!;
+
+
+        public Connection(IObjectContainer objectContainer)
+        {
+            this.objectContainer = objectContainer;
+        }
+
+
+        [BeforeTestRun]
+        public static void BeforeTestRun()
+        {
+            _webApplicationFactory = new CustomWebApplicationFactory<Program>();
+        }
+
+        [BeforeScenario]
+        public void InitializeClient()
+        {
+            var httpClient = _webApplicationFactory.CreateClient();
+            objectContainer.RegisterInstanceAs(httpClient);
```

Comment:
inject the httpClient in context of the tests. I am not sure this is the best way, but it works for now.

Relevant links:
https://docs.specflow.org/projects/specflow/en/latest/Bindings/Sharing-Data-between-Bindings.html
https://docs.specflow.org/projects/specflow/en/latest/Bindings/Context-Injection.html